pom.xml

```diff
@@ -29,8 +29,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <!-- DependencyGraph is final in Jenkins < 1.425 -->
-        <version>1.425</version>
+        <version>1.532.3</version>

```

Comment:
I found your commit comment "Use core that support java 1.6 and JenkinsRule".
I don't think that's a good idea to change the target version for the testing purpose.